src/kOS/Binding/TerminalSettings.cs

```diff
@@ -10,6 +11,7 @@ public override void AddTo(SharedObjects shared)
             shared.BindingMgr.AddGetter("SESSIONTIME", () => shared.Cpu.SessionTime);
             shared.BindingMgr.AddGetter("VERSION", () => Core.VersionInfo);
             shared.BindingMgr.AddGetter("VOLUME:NAME", () => shared.VolumeMgr.CurrentVolume.Name);
+            shared.BindingMgr.AddGetter("TERMINAL", () => new TerminalStruct(shared));
```

Comment:
Incedentally, why on earth were the above exiting 3 things in a file called "TerminalSettings"??  They have nothing to do with terminals.